### DISCUSSION TITLE
Hindi translation group

### DISCUSSION THREAD

**Discussion by jlooper** [microsoft/ML-For-Beginners#558] [Translations]
Gather here for feedback on your translations and to standardize vocabulary

**Comment by iamrahulmahato** [comment]
I have created PR #557 kindly review it and if there is any feedback please let me know.

**Comment by iamrobin123** [comment]
@iamrahulmahato translation is good,can I work on other file for hindi translation

**Reply by iamrahulmahato** [reply]
@iamrobin123 yes you can work on other file for hindi translation
